packages/protocol-contracts/contracts/interfaces/ZetaErrors.sol

```diff
@@ -1,16 +1,25 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.7;
 
+/**
+ * @dev Common custom errors
+ */
 interface ZetaErrors {
+    // @dev Error thrown when caller is not the address defined as TSS address
```

Comment:
Let's go for Thrown on all the error files instead of Error thrown (just to save a few words).